main.cpp

```diff
@@ -48,6 +58,13 @@ int main(int argc, char *argv[])
     }
 
     qDebug() << "Git version:" << GIT_VERSION;
+    LogSettings();
+    try {
```

Comment:
why?

Reply:
My build had some kind of linking issue where the exception handler didn't work. I put this here so it's easy to diagnose if the problem reoccurs.